aws/src/test/java/org/apache/iceberg/aws/s3/TestS3OutputStream.java

```diff
@@ -85,6 +87,8 @@
   private final Random random = new Random(1);
   private final Path tmpDir = Files.createTempDirectory("s3fileio-test-");
   private final String newTmpDirectory = "/tmp/newStagingDirectory";
+  private final List<Tag> tags = ImmutableList.of(
+      Tag.builder().key("abc").value("123").build());
```

Comment:
Request will be rejected in case we have duplicate keys: https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/allocation-tag-restrictions.html

Though, we are not supposed to have duplicate keys as the catalog `properties` map maintains uniqueness. So, if the user passes: 
```
--conf spark.sql.catalog.my_catalog.s3.write.tags.my_key=my_val \
--conf spark.sql.catalog.my_catalog.s3.write.tags.my_key=my_val2
```

then the `properties` map would store only `s3.write.tags.my_key=my_val2` which is later parsed to extract `my_key=my_val2` and then converted to `writeTags`.